lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -196,7 +274,12 @@ class ReportRenderer {
 }
 
 if (typeof module !== 'undefined' && module.exports) {
-  module.exports = ReportRenderer;
+  module.exports = {
```

Comment:
if these need to be exported just make them static members?

OTOH, if it's just for testing (I think it is right now?), I'd rather just test the effects on the final generated report so anyone doing future refactors isn't spending their time moving stuff around just for tests...the score is just correctly formatted or it's not